classes/Http/Form/SignupForm.php

```diff
@@ -60,19 +60,18 @@ public function validateAll($action = 'create')
             $valid_speaker_bio = $this->validateSpeakerBio();
         }
 
-        return (
-            $valid_email &&
-            $valid_passwords &&
-            $valid_first_name &&
-            $valid_last_name &&
-            $valid_company &&
-            $valid_twitter &&
-            $valid_url &&
-            $valid_speaker_info &&
-            $valid_speaker_bio &&
+        return
+            $valid_email         &&
```

Comment:
@mdwheele 

Regarding your question why these changes here have been made, I'll try to find out.